fhir_zulip_nlp/fhir_zulip_nlp.py

```diff
@@ -206,13 +216,16 @@ def create_report2(
             df_kw = df_kw.sort_values(['timestamp'])  # oldest first
             threads: List[str] = list(df_kw['subject'].unique())
             # Average thread length
+            # TODO: Refactor to pandas?
             tot_thread_len = 0
             for thread in range(len(threads)):
                 df_thread = df_kw[df_kw['subject'] == threads[thread]]
                 z = (list(df_thread['timestamp'])[-1] - list(df_thread['timestamp'])[0]) / seconds_per_day
                 tot_thread_len += float(f'{z:.1f}')
             avg_thread_len = round(tot_thread_len / len(threads), 3)
             # Outliers
+            # TODO: Refactor to pandas?
+            # TODO: Add cols for 1 and 2 std deviations?
```

Comment:
@rohaher Idk if others will care about this change, but I intend to change this to show just how many standard deviations away from thread length. Might need to make 3 columns for these outliers: std deviations away from (i) keyword avg, (ii) category avg, (iii) avg of all of our queried category/keyword threads.